ods_ci/tests/Resources/Files/llm/serving_runtimes/tgis_servingruntime_grpc.yaml

```diff
@@ -9,7 +9,7 @@ spec:
       name: pytorch
   containers:
     - name: kserve-container
-      image: quay.io/modh/text-generation-inference@sha256:18048121be7624d8cfe3f387e6de7ebb2e9376213f795d66cada26d8391229ca
+      image: quay.io/modh/text-generation-inference@sha256:2f2438084666ed6b2ad522ba76b825b4c0985e56fab25b4c10eed289e4a4832a
```

Comment:
how sustainable is this? We should find some other way of dealing with the versions in the yamls...

Reply:
agreed..planning to create github action to handle this in future automatically